src/controllers/apps/video-app/comment.controller.js

```diff
@@ -0,0 +1,256 @@
+import mongoose, { isValidObjectId } from "mongoose";
+import { Comment } from "../../../models/apps/video-app/comment.model.js";
+import { ApiError } from "../../../utils/ApiError.js";
+import { ApiResponse } from "../../../utils/ApiResponse.js";
+import { asyncHandler } from "../../../utils/asyncHandler.js";
+import { Video } from "../../../models/apps/video-app/video.model.js";
+import { getMongoosePaginationOptions } from "../../../utils/helpers.js";
+
+/**
+ * Retrieves the comments for a specific video.
```

Comment:
No need to add JS Docs for controllers we are not following this convention anywhere in the application